BrightID/src/components/HomeScreen.tsx

```diff
@@ -50,36 +55,26 @@ const discordUrl = 'https://discord.gg/nTtuB2M';
 
 /** Selectors */
 
-export const verifiedAppsSelector = createSelector(
-  selectAllApps,
+export const verificationsTextsSelector = createSelector(
   (state: State) => state.user.verifications,
-  (apps, userVerifications) => {
-    // check for each app if the user has at least one of the verifications
-    return apps.filter((app: AppInfo) => {
-      // ignore testing apps
-      if (app.testing) return false;
-      let hasOneVerification = false;
-      if (app.verifications) {
-        for (const verification of app.verifications) {
-          const verified = isVerified(
-            _.keyBy(userVerifications, (v) => v.name),
-            verification,
-          );
-          if (verified) {
-            hasOneVerification = true;
-          }
-        }
-      }
-      return hasOneVerification;
-    });
+  (verifications) => {
```

Comment:
Same code as in ConnectionScreenController. Should be moved to a utility function.